content/tutorials/publish-subscribe.textile

```diff
@@ -16,6 +16,56 @@ Messages published can contain string, JSON object, JSON array or binary data pa
 
 h2. Step 2 - Install Ably
 
+blang[java].
+  To start using Ably in your Android app, you need to include the Ably Client library. We recommend that you include the latest client library via Gradle in your module-level @gradle.build@ file.
```

Comment:
There is an entire step missing here.  Step 2 should be set up the Android environment with step-by-step instructions to get that set up.  This is meant to be a tutorial people can follow from start to finish without any prior knowledge, and we have assumed they are up and running already at this point.